src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => {
+  const partnersName = sex === 'm'
+    ? `${partnerName} is my wife`
+    : `${partnerName} is my husband`;
+  const hasPartner = isMarried === true
```

Comment:
you can avoid of creation of a new variable and use this condition inside layout.